### PR TITLE
redux impl

### PR DESCRIPTION
**Pull request by otuffaha**

### CODE REVIEW

File: src/reducers/RecipeRecommendationReducer.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { recipeRecommendationActionTypes as actionTypes } from '../constants/actionTypes';
```
Comment (seanyles) [inline src/reducers/RecipeRecommendationReducer.js]: This is probably fine, but it should be part of the existing RecipeReducer

**Review by seanyles — DISMISSED** [review]

File: src/reducers/RecipeRecommendationReducer.js (hunk 1 above)
Comment (mavalos1) [reply src/reducers/RecipeRecommendationReducer.js]: yes I agree, it should follow the same pattern

**Review by mavalos1 — CHANGES_REQUESTED** [review]
Only one tiny comment

File: src/reducers/RecipeRecommendationReducer.js (hunk 1 above)
Comment (otuffaha) [reply src/reducers/RecipeRecommendationReducer.js]: Does it matter?

Comment (otuffaha) [reply src/reducers/RecipeRecommendationReducer.js]: I would argue that it is the same

**Comment by mavalos1** [comment]
rebasing

**Review by mavalos1 — APPROVED** [review]

**Pull request merged**